### DISCUSSION TITLE
Student pack

### DISCUSSION THREAD

**Discussion by rockystar12345** [community/community#54503] [GitHub Education]
### Hi everyone,

I am a dropper for jee main exam . And not student pack .. as my all apply get cancelled.. how should I do .. I used to submit my jee main result all ... What should I do

**Comment by samus-aran** [comment]
Hi @rockystar12345  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
